ui/fields/datetime.php

```diff
@@ -154,12 +155,8 @@
 	jQuery( function ( $ ) {
 		var $container = $( '<div>' ).appendTo( 'body' ).addClass( 'pods-compat-container' ),
 			$element   = $( 'input#<?php echo esc_js( $attributes['id'] ); ?>' ),
-			beforeShow = {
-				'beforeShow': function( textbox, instance) {
-					$( '#ui-datepicker-div' ).appendTo( $container );
-				}
-			},
-			args = $.extend( <?php echo json_encode( $args ); ?>, beforeShow );
+			$alt       = null,
+			args       = <?php echo json_encode( $args ); ?>;
```

Comment:
:warning: json_encode() is discouraged. Use wp_json_encode() instead.

<sub>hash: 4c556675f5b28b8dce016292bfda7377</sub>